### PR TITLE
GCloud 9 supplier declaration

### PR DESCRIPTION
**Pull request by cathrooney**
This is a test

### CODE REVIEW

File: frameworks/g-cloud-9/manifests/declaration.yml [hunk 1]
```diff
@@ -1,41 +1,45 @@
 -
-  name: Essentials
+  name: Providing suitable services
   editable: True
   description: >
     You must make this declaration to apply to G-Cloud&nbsp;9. You can come back and change your answers at any time before
-    5pm GMT, 14 December 2016.
+    5pm BST, 11 April 2017. [might want to remove]
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/manifests/declaration.yml]: We probably at least want to remove the bit in square brackets... or leaving it in for discussion?

**Comment by TheDoubleK** [comment]
Did we lose any questions altogether yet? Or is that still up for discussion with CCS?

**Comment by TheDoubleK** [comment]
I've rebased it so the green "merge" button will work. This looks fine to merge if you're happy leaving the square bracket thing in for now?

Probably fine without a version number bump, we can bump the version once the service questions go in and see it in all its glory all together.

File: frameworks/g-cloud-9/manifests/declaration.yml (hunk 1 above)
Comment (cathrooney) [inline frameworks/g-cloud-9/manifests/declaration.yml]: Leaving it in for discussion

**Comment by cathrooney** [comment]
Haven't removed any questions yet as decision still pending with CCS Sourcing

**Comment by cathrooney** [comment]
Yes, happy to leave square brackets in - it was deliberate - need to reconsider that line with the new flow but didn't want to lose it

**Pull request merged**